client/pubspec.yaml

```diff
@@ -64,6 +64,9 @@ dev_dependencies:
   flutter_test:
     sdk: flutter
 
+dependency_overrides:
+  geolocator: ^13.0.1
```

Comment:
Would it make sense to bump `geolocator` package version in https://github.com/flet-dev/flet/blob/main/packages/flet_geolocator/pubspec.yaml#L15 rather than in the client app?

Reply:
Yeah actually that would be a better idea

Reply:
Done